src/libraries/System.Security.Cryptography.Algorithms/tests/HmacMD5Tests.cs

```diff
@@ -85,5 +85,11 @@ public void HMacMD5_Rfc2104_2()
         {
             VerifyHmacRfc2104_2();
         }
+
+        [Fact]
+        public void HMacMD5_ThrowsArgumentNullForNullConstructorKey()
+        {
+            Assert.Throws<ArgumentNullException>(() => new HMACMD5(null));
```

Comment:
```suggestion
            AssertExtensions.Throws<ArgumentNullException>("key", () => new HMACMD5(null));
```

(and similarly for the rest)

Reply:
Also, while it's slightly redundant, it's probably worth verifying the set_Key exception on each implementation type.

Reply:
> Also, while it's slightly redundant, it's probably worth verifying the set_Key exception on each implementation type.

Not sure I follow - are you suggesting making the same `AssertExtensions` change to to `Key` setter's test?

Reply:
Apparently I had a brain-o.  I read the last test as using `HMAC.Create()`, not a test-specific `Create()` from inheritance, so I thought we were only testing one algorithm for set_Key throwing.

But HMAC.Create() throws PNSE, so it clearly isn't... and I've now reconstructed what that test does.

::whistles innocently::